Sources/AEPOptimize/Optimize+PublicAPI.swift

```diff
@@ -22,8 +22,9 @@ public extension Optimize {
     /// - Parameter decisionScopes: An array of decision scopes.
     /// - Parameter xdm: Additional XDM-formatted data to be sent in the personalization request.
     /// - Parameter data: Additional free-form data to be sent in the personalization request.
-    @objc(updatePropositions:withXdm:andData:)
-    static func updatePropositions(for decisionScopes: [DecisionScope], withXdm xdm: [String: Any]?, andData data: [String: Any]? = nil) {
+    /// - Parameter completion: An optional callback to be used by consumer
+    @objc(updatePropositions:withXdm:andData:completion:)
+    static func updatePropositions(for decisionScopes: [DecisionScope], withXdm xdm: [String: Any]?, andData data: [String: Any]? = nil, _ completion: ((Bool, Error?) -> Void)? = nil) {
```

Comment:
Wondering if we can do away with the boolean value and simplify the completion handler to just `_ completion: ((Error?) -> Void)?`. If there's no error, we can return null. The boolean field seems a bit superfluous. Thoughts?

Reply:
We have changed this to keep list of successful decision scopes and AEPOptimize error as of now